src/google/protobuf/any.pb.h

```diff
@@ -129,12 +129,12 @@ class PROTOBUF_EXPORT Any final :
       const ::PROTOBUF_NAMESPACE_ID::FieldDescriptor** value_field);
   template <typename T, class = typename std::enable_if<!std::is_convertible<T, const ::PROTOBUF_NAMESPACE_ID::Message&>::value>::type>
   bool PackFrom(const T& message) {
-    return _any_metadata_.PackFrom<T>(message);
```

Comment:
NB: this also includes fixes for #8758. At the time I did not knew about this script for checked-in files regeneration.